### PR TITLE
CVP-3530: MSP / CVP - Reduce the number of ready/standby clusters in cluster pools

### PR DESCRIPTION
**Pull request by apodhrad**
Signed-off-by: Andrej Podhradsky <apodhrad@redhat.com>

### CODE REVIEW

File: clusters/hive/pools/msp/msp-ocp-4-11-medium-single-az-aws_clusterpool.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ spec:
       region: us-east-1
   pullSecretRef:
     name: pull-secret
-  size: 4
+  size: 0
```
Comment (danilo-gemoli) [inline clusters/hive/pools/msp/msp-ocp-4-11-medium-single-az-aws_clusterpool.yaml:35]: Shouldn't this be `1`?

Comment (apodhrad) [reply clusters/hive/pools/msp/msp-ocp-4-11-medium-single-az-aws_clusterpool.yaml:35]: No, our intention is to significantly reduce the resources on AWS for this project (MSP). Our priorities and plans were changed and there is no need to keep any hibernated cluster in MSP pools.

**Pull request merged**